CMakeLists.txt

```diff
@@ -45,7 +45,7 @@ option(BENCHMARK_ENABLE_LIBPFM "Enable performance counters provided by libpfm"
 set(CMAKE_CXX_VISIBILITY_PRESET hidden)
 set(CMAKE_VISIBILITY_INLINES_HIDDEN ON)
 
-if(MSVC)
+if(CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
```

Comment:
I'm assuming here that the below workaround is only needed for the "real" MSVC and not `clang-cl` or `clang++` as they can natively cross compile and don't have separate ARM variants AFAIK.

Reply:
that's my assumption too.